utils/zsh/patches/010-use-libpcre2.patch

```diff
@@ -0,0 +1,619 @@
+--- a/Src/Modules/pcre.c
```

Comment:
Hmm, this patch still requires some formalities. No one knows by looking into it that it is backport.

I know I could be annoying with this one, but it is in doc:
https://openwrt.org/submitting-patches#patch_checklist

and I really tired of patches, which do not have Git header. In main repo, there's plenty of them. Many of them hacks, which are really hard to be removed.

So, please you are not doing it only for me, but for others. It will be in Git history.

Reply:
There is no git header, since its source is not from git - only source code is from git. Surely as there is no official version with footprint of this source code, since it has code combined from both versions, this is a hack. Though when new version is released, removing this hack is as simple as removing patch.

Make a request change with requested change. For me this is completely self explanatory, but that's the point; I hardly ever read patch comments, instead I read the source code as C(++) is often simpler than descriptions that are nearly always vague for one who's mother language is not English.

But sure, make a change request with text of your desire.. And I'll apply it.